src/libpsl-native/src/nativesyslog.cpp

```diff
@@ -0,0 +1,17 @@
+#include <syslog.h>
+#include <nativesyslog.h>
+
+extern "C" void Native_SysLog(int32_t priority, const char* message)
```

Comment:
Many `cpp` files in the directory has "//! " comments. If it is best practice please add doc comments.

Reply:
fixed